content/en/docs/v3.5/quickstart.md

```diff
@@ -1,61 +1,122 @@
 ---
 title: Quickstart
 weight: 900
-description: Get etcd up and running in less than 5 minutes!
+description: Run etcd Locally Under 5 minutes!
 ---
 
-Follow these instructions to locally install, run, and test a single-member
-cluster of etcd:
+This guide walks through installing and running a single-node etcd cluster locally using pre-built binaries - no containers or cloud set up required.
 
- 1. Install etcd from pre-built binaries or from source. For details, see
-    [Install][].
+## Prerequisites
 
-    {{% alert color="warning" %}}**Important**: Ensure that you perform the last
-    step of the installation instructions to verify that `etcd` is in your path.
-    {{% /alert %}}
+Make sure you're on a supported platform before getting started:
 
- 2. Launch `etcd`:
+- Operating System: Ubuntu 22.04+ or macOS 11+. Windows users can use WSL2.
+- [Supported platforms architecture](/docs/v3.5/op-guide/)
+- Tools installed:
+  - `curl` and `tar`
+  - Internet connection to download the latest etcd [release](https://github.com/etcd-io/etcd/releases/).
+  - `sudo` access to install binaries.
 
-    ```console
-    $ etcd
-    {"level":"info","ts":"2021-09-17T09:19:32.783-0400","caller":"etcdmain/etcd.go:72","msg":... }
-    ⋮
-    ```
+## Steps
 
-    {{% alert color="info" %}}**Note**: The output produced by `etcd` are
-    [logs](../op-guide/configuration/#logging) &mdash; info-level logs can
-    be ignored. {{% /alert %}}
+### 1. Installation
```

Comment:
We have this dedicated page for install - https://etcd.io/docs/v3.5/install/ We should try to avoid dual maintenance. I would suggest to make the install page better and provide a link to it from here. That way if anything change with installation steps, we just need to make change to one place.